Library/Homebrew/constants.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 # RuboCop version used for `brew style` and `brew cask style`
-HOMEBREW_RUBOCOP_VERSION = "0.50.0"
```

Comment:
Can we put these in the `Gemfile` instead now?

Reply:
Not in this PR.